dds_registration/views/membership.py

```diff
@@ -41,7 +42,7 @@ def membership_application(request: HttpRequest):
                     "membership": {
                         "type": form.cleaned_data["membership_type"],
                     },
-                    "method": form.cleaned_data["payment_method"],
+                    "method": payment_method,
```

Comment:
What is the idea behind this change? You create a variable to only use it once. I suggest reverting this change.